codecov.yml

```diff
@@ -5,3 +5,6 @@ coverage:
     changes: false
 
 comment: off
+
+fixes:
+  - "::rest_framework/"
```

Comment:
btw: why are the statuses disabled?  Because of this issue?
I think it would be nice to have them reported via Github's status UI.